hack/images/dcos/Makefile

```diff
@@ -1,65 +1,153 @@
-mkfile_path     := $(abspath $(lastword $(MAKEFILE_LIST)))
-current_dir     := $(patsubst %/,%,$(dir $(mkfile_path)))
-
-FROM ?= $(current_dir)/_build/k8sm
-TARGET_OBJ = km kubectl
-SOURCE_OBJ = $(TARGET_OBJ:%=$(FROM)/%)
-
+# input variables
+GIT_REF ?= v1.0.2-v0.6.1
+GIT_REPO ?= mesosphere/kubernetes
```

Comment:
Yeah, I'm also surprised that we're using our fork for the build. As long as we define a GIT_REF (and hopefully that ref isn't overwritten in upstream), the we got reproducible builds.